servicenative/piccapautostart

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+luna-send -n 1 -f luna://org.webosbrew.piccap.service/isRunning '{}'
```

Comment:
I think we should just use `luna://org.webosbrew.piccap.service/start` here - it doesn't really matter whether we start it up via UI or during bootup, and just using `start` will make it much simpler on native service side IMO.

Reply:
I were thinking on the following: If in future we find a way in which the DBUS-Service will run automaticlly after startup, main is called and then executed the startup.
Now we are just triggering the service-startup using any method call to the service to call main. So if we are using the start method, we would do a double start, since of first calling main and then the start method.
Right now we could also process the return value, I think it is more the correct way, because for now it's only a workaround, until we find a better way of autostarting background services.
What do you think?